scheduler/src/cook/util.clj

```diff
@@ -190,3 +191,9 @@
   "Return triple of keys from two maps: [only in left, only in right, in both]"
   [left right]
   (data/diff (set (keys left)) (set (keys right))))
+
+(defn time-seq
```

Comment:
Split this refactor into a separate PR?

Reply:
Discussed offline the pros/cons of splitting this up, I decided to keep it as one PR